src/components/feature-details.vue

```diff
@@ -3,11 +3,10 @@
     class="feature-details"
     min-width="320"
     max-height="800"
-    v-if="hasData"
   >
     <v-toolbar dense flat>
       <v-toolbar-title>
-        {{ feature.properties.uid }}
+        {{ feature.uid }}
```

Comment:
The [data ](https://tools.ietf.org/html/rfc7946) model of a feature has `feature.properties` and `feature.id`, but not `feature.uid`, so `feature.properties.uid` was more accurate.

Reply:
What an unexpected pleasure. I would have dressed for the occasion if I knew this was going to be reviewed ☺️ . So previously I was passing the whole feature around, but this component really only needed two things: an `id` for the feature and an `id` for the layer the feature is part of. So this becomes more of a semantic discussion - `feature` is a bit of a misnomer here because it may indeed cause the impression that we're talking about the actual geojson feature object here. I will take this into account.